cmd/aries-agent-rest/startcmd/start.go

```diff
@@ -93,6 +93,18 @@ const (
 		" Possible values [http] [ws]. Defaults to http if not set." +
 		" Alternatively, this can be set with the following environment variable: " + agentOutboundTransportEnvKey
 
+	agentTLSCertFileFlagName      = "tls-cert-file"
+	agentTLSCertFileEnvKey        = "TLS_CERT_FILE"
+	agentTLSCertFileFlagShorthand = "c"
+	agentTLSCertFileFlagUsage     = "tls certificate file." +
+		" Alternatively, this can be set with the following environment variable: " + agentTLSCertFileEnvKey
+
+	agentTLSKeyFileFlagName      = "tls-key-file"
+	agentTLSKeyFileEnvKey        = "TLS_KEY_FILE"
+	agentTLSKeyFileFlagShorthand = "k"
+	agentTLSKeyFileFlagUsage     = "tls key file." +
+		" Alternatively, this can be set with the following environment variable: " + agentTLSKeyFileEnvKey
+
```

Comment:
Also, we would need a configuration to trust the certs (eg, self-signed for testing). I think we will face this issue when we have the bdd tests when 2 agents interact. cc: @fqutishat

Reply:
So how did it work in @soluchok 's video?

Reply:
Ah - they were talking through a mediator.